backend/controllers/orgUsersController.js

```diff
@@ -32,6 +32,24 @@ const createOrgUser = async (req, res) => {
   }
 }
 
+const findOrgUserById = async (req, res) => {
+  const orgUserId = req.params.org_user_id;
+  try {
+    const orgUser = await OrgUser.find({_id:orgUserId});
```

Comment:
` OrgUser.find({_id:orgUserId}) I think it's ` OrgUser.findById({_id:orgUserId})`, it's .findById to find a specific id (I came across this yesterday when I was doing comments and finding the postID -_-

Reply:
yes apologies i have changed this in the current branch i am working on